features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/system-api-resource.xml

```diff
@@ -291,70 +345,81 @@
+            <Scope displayName="View Keystore" name="internal_keystore_view" 
+                    description="View certificates from the keystrore and truststore"/>
+            <Scope displayName="Update Keystore" name="internal_keystore_update" 
+                    description="Update certificates from the keystrore and truststore"/>
         </Scopes>
     </APIResource>
     <APIResource name="Notification Sender Management API"
                  identifier="/api/server/v1/notification-senders" requiresAuthorization="true"
                  description="API representation of the Notification Sender Management API" type="TENANT_ADMIN">
         <Scopes>
-            <Scope displayName="View Notification Senders" name="internal_notification_senders_view"/>
-            <Scope displayName="Create Notification Senders"
-                   name="internal_notification_senders_create"/>
-            <Scope displayName="Update Notification Senders"
-                   name="internal_notification_senders_update"/>
-            <Scope displayName="Delete Notification Senders"
-                   name="internal_notification_senders_delete"/>
+            <Scope displayName="View Notification Senders" name="internal_notification_senders_view" 
+                    description="View new notification senders for the organization (root)"/>
+            <Scope displayName="Create Notification Senders" name="internal_notification_senders_create" 
+                    description="Create notification senders for the organization (root)"/>
+            <Scope displayName="Update Notification Senders" name="internal_notification_senders_update" 
+                    description="Update notification senders for the organization (root)"/>
+            <Scope displayName="Delete Notification Senders" name="internal_notification_senders_delete" 
+                    description="Delete notification senders for the organization (root)"/>
         </Scopes>
     </APIResource>
     <APIResource name="Notification Sender Management API"
                  identifier="/o/api/server/v1/notification-senders" requiresAuthorization="true"
                  description="API representation of the Notification Sender Management API" type="ORGANIZATION_ADMIN">
         <Scopes>
-            <Scope displayName="View Notification Senders" name="internal_notification_senders_view"/>
-            <Scope displayName="Create Notification Senders"
-                   name="internal_notification_senders_create"/>
-            <Scope displayName="Update Notification Senders"
-                   name="internal_notification_senders_update"/>
-            <Scope displayName="Delete Notification Senders"
-                   name="internal_notification_senders_delete"/>
+            <Scope displayName="View Notification Senders" name="internal_org_notification_senders_view" 
+                    description="View new notification senders for organization"/>
```

Comment:
based on how the description has been formed for other APIs, we shouldn't have `new` for view, update and delete descriptions right?

Reply:
Removed